.pre-commit-config.yaml

```diff
@@ -25,15 +25,16 @@ repos:
           - --autofix
 
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.0.286
+    rev: v0.0.289
     hooks:
       - id: ruff
         types_or: [python, pyi, jupyter]
         args:
           - --fix
           - --target-version=py311
-          - --select=B,C,E,F,W
+          - --select=B,C,E,F,W,D
```

Comment:
The `D` option makes it so ruff looks at documentation: https://docs.astral.sh/ruff/rules/#pydocstyle-d